.github/workflows/arm64.yml

```diff
@@ -0,0 +1,64 @@
+name: arm64
+on:
+  workflow_dispatch:
+    inputs:
+      plugins:
+        description: "Plugins to build (i.e. connect-go:latest, connect-go, grpc/java:v1.53.0)"
+        default: ''
+        type: string
+
+env:
+  DOCKER_ORG: "ghcr.io/bufbuild"
+
+# Ref: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: read
+  contents: read
+#  packages: write
+
+concurrency: arm64-${{ github.ref }}
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  arm64:
+    runs-on: ubuntu-latest-4-cores
+    steps:
+    - name: set PLUGINS from workflow inputs
+      if: ${{ inputs.plugins }}
+      run: echo "PLUGINS=${{ inputs.plugins }}" >> $GITHUB_ENV
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Install Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.21'
+        check-latest: true
+    - name: Install buf cli
+      uses: bufbuild/buf-setup-action@v1
+      with:
+        github_token: ${{ github.token }}
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@2b82ce82d56a2a04d2637cd93a637ae1b359c0a7
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@885d1462b80bc1c1c7f0b00334ad271f09369c55
+    - name: Login to Docker Hub
+      uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+#    - name: Login to GitHub Container Registry
+#      if: github.repository == 'bufbuild/plugins'
+#      uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc
+#      with:
+#        registry: ghcr.io
+#        username: ${{ github.actor }}
+#        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Test
+      run: make test
+      env:
+        DOCKER_BUILD_EXTRA_ARGS: '--platform linux/arm64'
```

Comment:
This should be enough to build everything for arm64. There may be some tweaks we can make (especially to Go builds) to make the builds a ton faster.